### DISCUSSION TITLE
Zotero Integration

### DISCUSSION THREAD

**Discussion by hasantahir** [glushchenko/fsnotes#1385] [General]
Hi,

I have been enjoying the app for around a year now. Would it be possible to have a Zotero integration for research papers into FS Notes? Zettlr, an electron-based note-taking app has a [similar feature](https://docs.zettlr.com/en/academic/citations/). Or perhaps any suggestion for a method to keep track of research papers through the already available toolset of FS Notes.

Cheers.

**Comment by glushchenko** [comment]
Not planned, sorry.

**Reply by hasantahir** [reply]
Thanks and looking forward to v6.
